src/parser/jobs/sam/modules/Kenki/Kenki.tsx

```diff
@@ -30,14 +31,10 @@ export class Kenki extends CoreGauge {
 	@dependency private suggestions!: Suggestions
 
 	private kenkiGauge = this.add(new CounterGauge({
-		chart: {label: 'Kenki Gauge', color: JOBS.SAMURAI.colour},
-		/*
 		graph: {
-			handle: 'Kenki',
 			label: <Trans id="sam.gauge.resource.kenkiLabel">Kenki</Trans>,
-			color: JOBS.SAMURAI.colour,
+			color: Color(JOBS.SAMURAI.colour).fade(.25),
```

Comment:
Did you just cheat the linter on magic numbers by not having 0.25? bruh

Reply:
No longer magic.